templates/payment/form.php

```diff
@@ -16,29 +16,24 @@
 	<?php endif; ?>
 
 	<div>
-		<?php if ( $showExistingCards ) : ?>
+		<?php if ($showExistingCards) : ?>
 			<input id="new_card_info" type="radio" name="card_id" value="" />
 			<label id="label-new_card_info" for="new_card_info">
-				<h3><?php _e( 'Create a charge using new card', 'omise' ); ?></h3>
+				<h3><?php _e('Create a charge using new card', 'omise'); ?></h3>
 			</label>
 		<?php endif; ?>
 
-		<fieldset class="omise-new-card-form <?php echo $showExistingCards ? 'card-exists' : ''; ?>">
-
-			<?php require_once( 'form-creditcard.php' ); ?>
-
+		<div class="omise-new-card-form <?php echo $showExistingCards ? 'card-exists' : ''; ?>">
+			<div id="omise-card" style="width:100%; max-width: 400px;"></div>
+			<input type="hidden" name="omise_save_customer_card" class="omise_save_customer_card" />
 			<div class="clear"></div>
-
-			<?php if ( $viewData['user_logged_in'] ) : ?>
-				<div class="omise-form-child omise-remember-card">
-					<input type="checkbox" name="omise_save_customer_card" id="omise_save_customer_card" />
-					<label for="omise_save_customer_card" class="inline">
-						<?php _e( 'Remember this card', 'omise' ); ?>
-					</label>
-				</div>
-			<?php endif; ?>
-
-			<div class="clear"></div>
-		</fieldset>
+		</div>
 	</div>
 </div>
+
+<script>
+	window.CARD_FORM_THEME = "<?php echo $viewData['card_form_theme'] ?>";
+	window.CARD_BRAND_ICONS = JSON.parse(`<?php echo json_encode($viewData['card_icons']) ?>`);
+	window.FORM_DESIGN = JSON.parse(`<?php echo json_encode($viewData['form_design']) ?>`);
+	window.LOCALE = `<?php echo get_locale(); ?>`;
```

Comment:
It would nice if we could use `<?= ?>` instead of `<?php echo ?>

Reply:
As a default `<?=` is not available on php. 
must be enable Short Tags in `php.ini` to use this.

Reply:
Ah right. Thanks.